submissions/adeyemiJuwon/HTMLforms/contact.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>contact</title>
+</head>
+<body>
+  <header>
+    <div>
+        <input type="search" name="q" value="" aria-label="Search" placeholder="Search">
```

Comment:
@Samju200 Why is the `name` attribute on the input field `q`? Update that.

Reply:
> @Samju200 Why is the `name` attribute on the input field `q`? Update that.

Okay
Done